Assets/MixedRealityToolkit.SDK/Features/Utilities/PointerUtils.cs

```diff
@@ -144,6 +144,159 @@ public static IEnumerable<IMixedRealityPointer> GetPointers()
                 }
             }
         }
-    }
 
+        /// <summary>
+        /// Gets the physics ray based on the interaction enabled pointer assosiciated with the head gaze.
+        /// </summary>
+        /// <param name="ray">The physics ray for the pointer</param>
+        /// <returns>
+        /// True if the ray contains valid data, false otherwise.
+        /// </returns>
+        public static bool TryGetHeadGazeRay(out Ray ray)
+        {
+            return TryGetRay(InputSourceType.Head, Handedness.None, out ray);
+        }
+
+        /// <summary>
+        /// Gets the physics ray based on the interaction enabled pointer assosiciated with the eye gaze.
+        /// </summary>
+        /// <param name="ray">The physics ray for the pointer</param>
+        /// <returns>
+        /// True if the ray contains valid data, false otherwise.
```

Comment:
I think it would be good to explain what 'valid data' actually means here. Here's what I would expect:

True if eye gaze is available, false otherwise. Eye gaze is available if a platform both supports eye gaze (for example, MRTK is running on HoloLens 2) AND the user has calibrated their eyes. If a user's eye gaze is not available, this method returns false and the output ray is set to null / empty / zero.

Reply:
good suggestion

Reply:
what if i phrase it as "can be retrieved"?